editoast/openapi.yaml

```diff
@@ -3349,11 +3473,68 @@ components:
         status:
           type: integer
           enum:
-          - 401
+          - 403
+        type:
+          type: string
+          enum:
+          - editoast:authz:Unauthorized
+    EditoastAuthzErrorAuthError:
```

Comment:
I understand it would probably imply significant modification, not in the scope of this PR. I'm wondering if there isn't an easy path forward, renaming all enums' variants that are named something like `AuthorizationError::AuthError` into `AuthorizationError::Auth` since we most likely will never use the variant without the enum's name and therefore, we will know it's about an error in the code?

Reply:
Removed the `Error` suffix.